code/modules/jobs/job_types/shaft_miner.dm

```diff
@@ -8,6 +8,7 @@
 	spawn_positions = 3
 	supervisors = "the quartermaster and the head of personnel"
 	selection_color = "#dcba97"
+	roundstart_experience = list(/datum/skill/mining = SKILL_EXP_JOURNEYMAN) //Miners know their shit
```

Comment:
let's try this w/o so we can see how """early game""" looks like

Reply:
ok